cassandra/connection.py

```diff
@@ -61,18 +61,22 @@
 except ImportError:
     pass
 else:
+    # Old versions of lz4 has not block attribute.
+    # In new versions need to use lz4.block, because functions from
+    # lz4 namespace are deprecated.
+    lz4_block = lz4.block if hasattr(lz4, "block") else lz4
```

Comment:
@vetal4444 Can you make the change I mentioned in the PR conversation about this? Otherwise, LGTM.